cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -267,6 +258,22 @@
     } );
 }
 
+.m-global-search {
+    // Tab trigger is used to capture press of the tab key so that
+    // global search can be collapsed when it hits this element.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/2c8ab804eaa09543b3a569e93172906825e5c85b